tests/test_crud_immunisation_api.py

```diff
@@ -270,18 +278,25 @@ def test_bad_nhs_number_nhs_login(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
         "login_form": {"username": "656005750104"},
     }
 )
-@pytest.mark.parametrize("nhs_number,is_restricted", [(valid_nhs_number_with_s_flag, True), (valid_nhs_number1, False)])
-def test_get_s_flag_patient(nhsd_apim_proxy_url, nhsd_apim_auth_headers, nhs_number, is_restricted):
+@pytest.mark.parametrize(
+    "nhs_number,is_restricted",
+    [(valid_nhs_number_with_s_flag, True), (valid_nhs_number1, False)],
+)
+def test_get_s_flag_patient(
+    nhsd_apim_proxy_url, nhsd_apim_auth_headers, nhs_number, is_restricted
+):
     # Arrange
     token = nhsd_apim_auth_headers["Authorization"]
     imms_api = ImmunisationApi(nhsd_apim_proxy_url, token)
 
     # Act
     # TODO: Replace this with the usual specification example when it is updated.
```

Comment:
This TODO is part of this ticket. By "usual" I mean the `create_an_imms_obj` function.

Reply:
Data has been updated, will remove the TODO.

Reply:
Already done, (the data in the json file has been updated), just forgot to remove the to-do. TODO duly deleted

Reply:
Sorry, I mean that all the other tests use the examples from the specification directory (or should do), as `create_an_imms_obj` does. I didn't in this test because I didn't want to mess with the spec which was being actively worked on. Now the example should match the code so this can use `create_an_imms_obj`. If it doesn't work then one or the other of this test or the spec needs to change.

Reply:
We've changed it to use `create_an_imms_obj` and made sure all practitioner records are checked for s flag redactions where necessary